swan-lake/learn/observing-ballerina-code.md

```diff
@@ -37,10 +37,10 @@ Prometheus, etc. You can follow [Docker documentation](https://docs.docker.com/i
 
 ### Step 2 - Installing and Configuring the External Systems
 
-* Setup Prometheus for collecting metrics information by following section on [Setting up Prometheus](#prometheus)
-* Setup Grafana to visualize metrics by following section on [Setting up Grafana](#grafana)
-* Setup Jaeger analyze tracing as mentioned in section [Setting up Jaeger](#jaeger-server)
-* Setup Elastic Stack only if you are interested in analysing logs by following section on [Setting up Elastic Stack](#elastic-stack)
+* Setup Prometheus for collecting metrics information by following section on [Setting up Prometheus](#setting-up-prometheus)
```

Comment:
```suggestion
* Set up Prometheus for collecting metrics information by following the section on [Setting up Prometheus](#setting-up-prometheus)
```

Reply:
Fixed in ad688a6a3d97f0d8b4988ad4d3f624ee746d91b2